### ISSUE TITLE
Make touch capacitance configurable from Micropython

### ISSUE THREAD

**Issue by josuah** [Itsbrilliantlabs/monocle-micropython#5]
Different people have different body capacitance, so various sensitivity might help compensating different contexts.
I.e. "gloves mode".

**Comment by vr2045** [comment]
Good point. We really need a lot of testing for this. There are only 2-3 parameters the user needs control over in the AZ chip to do this tuning. They will require the AZ registers also to be read so the current value of the prox reading can be printed to the screen for setting the threshold.
